ee/cis/macos-26/test/scripts/CIS_2.6.8_pass.sh

```
@@ -0,0 +1,57 @@
+  "system.preferences.energysaver"
+  "system.preferences.network"
+  "system.preferences.printing"
+  "system.preferences.sharing"
+  "system.preferences.softwareupdate"
+  "system.preferences.startupdisk"
+  "system.preferences.timemachine"
+)
+
+for section in "${authDBs[@]}"; do
+  plist="/tmp/$section.plist"
+  /usr/bin/sudo /usr/bin/security -q authorizationdb read "$section" > "$plist"
```

Shellcheck (0.11.0)

[warning] sudo doesn't affect redirects. Use ..| sudo tee file

(SC2024)